chef/cookbooks/crowbar/templates/default/crowbar-pre-upgrade.sh.erb

```diff
@@ -92,25 +92,25 @@ zypper --non-interactive up <%= @neutron_agent %>
 crudini --set /etc/neutron/neutron-openvswitch-agent.conf.d/200-crowbar-upgrade.conf ovs ovsdb_interface vsctl
 crudini --set /etc/neutron/neutron-openvswitch-agent.conf.d/200-crowbar-upgrade.conf ovs of_interface ovs-ofctl
 <% end %>
-systemctl start <%= @neutron_agent %>
+systemctl restart <%= @neutron_agent %>
```

Comment:
I assume the point @aplanas was raising is that we should not restart the service that is handled by pacemaker. For other cases, restarting the agent service as (it is currently in this script) should be correct